packages/core/admin/admin/src/pages/MarketplacePage/components/EmptyPluginSearch/EmptyPluginGrid.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Box } from '@strapi/design-system/Box';
+
+const EmptyPluginCard = styled(Box)`
+  background: linear-gradient(180deg, rgba(234, 234, 239, 0) 0%, #eaeaef 100%);
+  opacity: 0.33;
+`;
+
+const GridColSize = {
+  S: 180,
+  M: 250,
+};
+
+const PlaceholderSize = {
+  S: 138,
+  M: 234,
+};
+
+const GridLayout = styled(Box)`
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(${({ size }) => `${GridColSize[size]}px`}, 1fr));
+  grid-gap: ${({ theme }) => theme.spaces[4]};
+`;
```

Comment:
Could you try using Grid and GridItem from the design system instead?

Reply:
How about GridLayout: https://design-system-git-main-strapijs.vercel.app/?path=/docs/design-system-components-gridlayout--base ?